src/features/activityFeed/DatasetActivityFeed.tsx

```diff
@@ -31,7 +31,8 @@ const DatasetActivityFeed: React.FC<DatasetActivityFeedProps> = ({
 
   useEffect(() => {
     dispatch(loadDatasetLogs(qriRef))
-  },[dispatch, qriRef])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
okie dokie

so this infinite loop happens because the `qriRef` object changes with every render. the way we have solved this in other places is rather than using `qriRef` in the dependency array, we use `qri.username` and `qri.name`, and any other parts of the ref that we need to respond to. we then create a new `qriRef` to pass into the dispatched method. 

Also, we seem to have two `useEffect` methods that are calling the same function. 

I think we should consolidate like this:

```ts
useEffect(() => {
    dispatch(loadDatasetLogs({username: qriRef.username, name: qriRef.name})
}, [dispatch, qriRef.username, qriRef.name, latestRun?.status]
```

untested code, but that should remove the infinite loop and keep us kosher with the dependency array!